config/manifests/bases/oadp-operator.clusterserviceversion.yaml

```diff
@@ -14,6 +14,8 @@ metadata:
     operatorframework.io/cluster-monitoring: "true"
     operatorframework.io/suggested-namespace: openshift-adp
     operators.openshift.io/infrastructure-features: '["Disconnected"]'
+    operators.openshift.io/valid-subscription: '["OpenShift Container Platform", "OpenShift
+      Platform Plus"]'
```

Comment:
why new line?

Reply:
because
https://github.com/openshift/oadp-operator/blob/726af6350789e868c029b870007deba888ff9bdc/Makefile#L262
automatically caused it to truncate the line.